### ISSUE TITLE
ordering is messed up on diff reports

### ISSUE THREAD

**Issue by catdad** [catdad/grandma#123]
This command maintains the order from the command line to the report:

```
grandma diff -l huge.log huge-dcs.log huge-dcs-sss.log
```

```
huge.log: postToViewingSessionToAllPages, 10m, 18 concurrent, 369 total

huge-dcs.log: postToViewingSessionToAllPages, 10m, 18 concurrent, 324 total

huge-dcs-sss.log: postToViewingSessionToAllPages, 10m, 18 concurrent, 302 total
```

This command gets the wrong values:

```
grandma diff -l huge25.log huge25-dcs.log huge25-dcs-sss.log
```

```
huge25.log: postToViewingSessionToAllPages, 10m, 25 concurrent, 344 total

huge25-dcs-sss.log: postToViewingSessionToAllPages, 10m, 25 concurrent, 379 total

huge25-dcs.log: postToViewingSessionToAllPages, 10m, 25 concurrent, 402 total
```

**Comment by stezu** [comment]
I think this is an issue with `async.mapValues` [here](https://github.com/catdad/grandma/blob/master/lib/report-diff.js#L148). The [docs](http://caolan.github.io/async/docs.html#mapValues) state the following:

> Note, the order of the keys in the result is not guaranteed. The keys will be roughly in the order they complete, (but this is very engine-specific)

**Comment by catdad** [comment]
Fixed in v0.6.4.
